docs/site/home.html

```diff
@@ -15,8 +15,11 @@ <h1 class="logo"><img src="site/img/logo-full.svg" alt="Google Cloud Platform" /
       </div>
 
       <div class="quote-box--supplementary">
-        <h2>One-line npm install</h2>
```

Comment:
@jgeewax look what I found:

![screen shot 2015-05-19 at 2 46 44 pm](https://cloud.githubusercontent.com/assets/723048/7711028/e578d66a-fe35-11e4-83f5-77f3271047a7.png)

Somewhere along the way, the font color must have been changed to match the background. Or, it was never visible from the start.

Python is also a victim...

![screen shot 2015-05-19 at 2 47 34 pm](https://cloud.githubusercontent.com/assets/723048/7711058/0b0d7660-fe36-11e4-82ab-86e88b6932a4.png)

... while Ruby doesn't seem to have it. I don't think the line is necessary, what do you think?